src/App.js

```diff
@@ -1,28 +1,17 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react'
```

Comment:
I believe semicolons should be used. Check out https://eslint.org/docs/rules/semi